frontend/src/assets/styles/_accordion.scss

```diff
@@ -0,0 +1,51 @@
+.accordion {
```

Comment:
Most of those CSS rules are available on Tachyons, would it be possible to use them on the component and only define the ones that are missing? I believe only the animation classes are missing. The font size of the accordion button could be a bit small (1rem / f6 on tachyons)